### PR TITLE
any-season border patrol grammar fixes

### PR DESCRIPTION
**Pull request by tobes03**
More grammar and syntax corrections applied to any-season border patrols in all the different biomes. Also, lots of {} added. Other files were affected by the following changes:

- Changing "judgement" to "judgment" across the board to keep with American spellings elsewhere
- Replacing all instances of "your patrol" with "the patrol"
- Replacing all instances of "cart" with "carry"
- Removed all instances of "altruistic"

### CODE REVIEW

File: resources/dicts/patrols/desert/training/any.json [hunk 1]
```diff
@@ -153,7 +153,7 @@
                 "weight": 5
             },
             {
-                "text": "s_c decides there has to be a smarter way of figuring out if a badger has moved in, one that doesn't involve sticking their head in front of a predator. They lead the patrol on a search of the scrubland around the sett, and come up with fresh badger scat, stinky, horrible, and perfect proof of the new occupant.",
+                "text": "s_c decides there has to be a smarter way of figuring out if a badger has moved in, one that doesn't involve sticking their head in front of a predator. They lead the patrol on a search of the scrubland around the sett, and come up with fresh badger scat, stinky, horrible, and incontrovertible of the new occupant.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/desert/training/any.json]: I'm a little wary of "incontrovertible" being too large a word.  I'll admit, I don't think I'd actually ever seen it before xD though I do understand the meaning.  Generally, though, we've tried not to get too complex with our vocabulary (it's a fine line, admittedly, I remember we had quite the discussion about "antagonize" as a button).

File: resources/dicts/patrols/wetlands/med/medcat_wetlands.json
Comment (scribblecrumb) [inline resources/dicts/patrols/wetlands/med/medcat_wetlands.json:1]: This file was removed recently in  #2258, I think you may have accidentally added it back?

File: resources/dicts/patrols/forest/border/border.json
Comment (scribblecrumb) [inline resources/dicts/patrols/forest/border/border.json:1]: Unsure if this is an accidental inclusion, in  #2258 the border/border.json file was renamed to border/any.json

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

File: resources/dicts/patrols/desert/training/any.json (hunk 1 above)
Comment (scribblecrumb) [reply resources/dicts/patrols/desert/training/any.json]: could use @Tiritirimatangi's opinion on the matter tbh

File: resources/dicts/patrols/forest/border/border.json
Comment (tobes03) [reply resources/dicts/patrols/forest/border/border.json:1]: Oh, I think I must have changed it back when I synced the forks. ~Should I rename border.json to any.json? There's an existing any.json in the forest/border folder.~

I just deleted the medcat_wetlands file and the redundant border.json file! Everything should check out now :) Thanks for catching the mistakes

File: resources/dicts/patrols/desert/training/any.json (hunk 1 above)
Comment (tobes03) [reply resources/dicts/patrols/desert/training/any.json]: Oh, I get that for sure. I can just revert it to "perfect proof!" I'm also realizing I mistakenly deleted the word "proof," so I should fix that either way.

Comment (Tiritirimatangi) [reply resources/dicts/patrols/desert/training/any.json]: Lol I don't mind either way occasionally I use weird word choices XD

Comment (tobes03) [reply resources/dicts/patrols/desert/training/any.json]: ❤️❤️ I did decide to switch it back to "perfect proof." everything should be good to go now!

**Review by scribblecrumb — APPROVED** [review]

**Comment by Thlumyn** [comment]
Me, an American for all her life, learning now that "judgment" is the proper US spelling. lol

File: resources/dicts/patrols/desert/border/any.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
                 ]
             },
             {
-                "text": "s_c sets up an ambush for the wandering dog, bloodying its snout and leading it on a wild chasing dance across the border and out of c_n territory.",
+                "text": "s_c sets up an ambush for the wandering dog. When the dog nears {PRONOUN/s_c/poss}'s hiding spot, s_c leaps out with claws unsheathed, bloodies its snout, and leads it on a wild chasing dance across the border and out of c_n territory.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/desert/border/any.json]: {PRONOUN/s_c/poss}'s
possessive and 's seems redundant

File: resources/dicts/patrols/forest/border/any.json [hunk 3]
```diff
@@ -464,7 +464,7 @@
                 ]
             },
             {
-                "text": "s_c decides there has to be an easy way to figure out if a badger has moved in, one that doesn't involve sticking head in front of a predator. They lead the patrol on a search of the clearing around the sett until they come upon some fresh badger scat - stinky, horrible, and perfect proof of the new occupant of the sett.",
+                "text": "s_c decides there has to be an easy way to figure out if a badger has moved in, one that doesn't involve sticking anyone's head in front of a predator. They lead the patrol on a search of the clearing around the sett until they come upon some fresh badger scat - stinky, horrible, and perfect proof proof of the new occupant of the sett.",
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/border/any.json]:
```suggestion
                "text": "s_c decides there has to be an easy way to figure out if a badger has moved in, one that doesn't involve sticking anyone's head in front of a predator. They lead the patrol on a search of the clearing around the sett until they come upon some fresh badger scat - stinky, horrible, and perfect proof of the new occupant of the sett.",
```

File: resources/dicts/patrols/forest/border/any.json [hunk 4]
```diff
@@ -619,7 +619,7 @@
...
```
Comment (Thlumyn) [inline resources/dicts/patrols/forest/border/any.json]:
```suggestion
                "text": "s_c decides there has to be an easy way to figure out if a badger has moved in, one that doesn't involve sticking anyone's head in front of a predator. They lead the patrol on a search of the clearing around the sett until they come upon some fresh badger scat - stinky, horrible, and perfect proof of the new occupant of the sett.",
```

**Review by Thlumyn — CHANGES_REQUESTED** [review]

**Review by Thlumyn — APPROVED** [review]

**Pull request merged**